config/config/sylius_fixtures.php

```diff
@@ -159,6 +159,23 @@
                     ],
                     'enabled' => true,
                 ],
+                'visa_mobile' => [
+                    'code' => 'visa_mobile',
+                    'name' => 'Visa mobile (Tpay)',
+                    'gatewayFactory' => 'tpay',
+                    'gatewayName' => 'tpay',
+                    'gatewayConfig' => [
```

Comment:
rebase and adjust setting the `gatewayConfig` accordingly to @lchrusciel changes; it's a simple thing